snow/engine/common/engine.go

```diff
@@ -427,3 +429,8 @@ type InternalHandler interface {
 	// Notify this engine of a message from the virtual machine.
 	Notify(context.Context, Message) error
 }
+
+type SimplexHandler interface {
+	// Notify this engine of a simplex message from nodeID.
+	SimplexMessage(ctx context.Context, nodeID ids.NodeID, msg *p2p.Simplex) error
```

Comment:
nit: All of these functions are messages, is there a reason we suffix this function with `Message`?